presto-main/src/test/java/com/facebook/presto/cost/ColumnStatisticsAssertion.java

```diff
@@ -91,4 +91,17 @@ public ColumnStatisticsAssertion distinctValuesCountUnknown()
                 "expected unknown distinctValuesCount but got " + statistics.getOnlyRangeColumnStatistics().getDistinctValuesCount().getValue());
         return this;
     }
+
+    public ColumnStatisticsAssertion dataSize(double expected)
+    {
+        assertEquals(statistics.getOnlyRangeColumnStatistics().getDataSize().getValue(), expected, "dataSize mismatch");
```

Comment:
shouldn't we have some notion of accuracy when comparing doubles? Often doubles will be "almost" the same since expected would be constant and actual would be computed